nrel/hive/model/vehicle/mechatronics/powercurve/powercurve_ops.py

```diff
@@ -1,37 +1,41 @@
+import logging
+
 from nrel.hive.model.energy.charger import Charger
 from nrel.hive.model.vehicle.vehicle import Vehicle
 from nrel.hive.model.vehicle.mechatronics import MechatronicsInterface
 from nrel.hive.util import Seconds, Ratio
 
-
+log = logging.getLogger(__file__)
 def time_to_full(
     vehicle: Vehicle,
     mechatronics: MechatronicsInterface,
     charger: Charger,
     target_soc: Ratio,
     sim_timestep_duration_seconds: Seconds,
+    min_delta_energy_change: Ratio = 0.0001
```

Comment:
looks great. to set this default value, could you:
- pass this default in from [here](https://github.com/NREL/hive/blob/main/nrel/hive/config/sim.py#L20) 
- everywhere it's called, for example [here](https://github.com/NREL/hive/blob/main/nrel/hive/dispatcher/instruction_generator/assignment_ops.py#L289), you could grab it from the environment (see below for example)
- default configuration is set in the [default hive.yaml](https://github.com/NREL/hive/blob/main/nrel/hive/resources/defaults/hive_config.yaml#L13) file, you can add the row `min_delta_energy_change: 0.0001` after schedule_type.

```python
min_delta_energy_change = env.config.sim.min_delta_energy_change
```

Reply:
Done

Reply:
done